lib/ansible/modules/system/lvol_thin.py

```diff
@@ -29,9 +27,9 @@
 DOCUMENTATION = '''
 ---
 author:
-    - "Pavol Ipoth <pavol.ipoth@gmail.com>"
```

Comment:
It should be indented by two spaces only like in the rest of the `DOCUMENTATION` block.